packages/compass-e2e-tests/tests/collection-export.test.ts

```diff
@@ -135,11 +141,324 @@ describe('Collection export', function () {
     expect(telemetry.screens()).to.include('export_modal');
   });
 
-  it('supports full collection to CSV');
-  it('supports collection to CSV with all fields');
-  it('supports collection to CSV with a subset of fields');
-  it('supports collection to JSON with a query filter');
-  it('supports full collection to JSON');
-  it('supports collection to JSON with all fields');
-  it('supports collection to JSON with a subset of fields');
+  it('supports full collection to CSV with all fields', async function () {
+    const telemetryEntry = await browser.listenForTelemetryEvents(telemetry);
+
+    // we're not going to use this query, but we're running it so we can make
+    // sure it gets ignored
```

Comment:
Just for my own understanding, this is “ignored” as in “not used as a filter for exported data”, not “ignored”  as in “not displayed in the export modal”, right?

Reply:
Gets displayed (few lines lower I think), but not used becuase we're selecting the other radio button.

Reply:
ie. That matches just one doc but we're going to make sure all 1000 got exported.